src/sql/src/func.rs

```diff
@@ -3161,6 +3161,10 @@ pub static PG_CATALOG_BUILTINS: LazyLock<BTreeMap<&'static str, Func>> = LazyLoc
             }) => String, 3538;
             params!(Bytes, Bytes) => Operation::binary(|_ecx, _l, _r| bail_unsupported!("string_agg on BYTEA")) => Bytes, 3545;
         },
+        "string_to_array" => Scalar {
+            params!(String, Any) => VariadicFunc::StringToArray => ScalarType::Array(Box::new(ScalarType::String)), 3947;
+            params!(String, Any, String) => VariadicFunc::StringToArray => ScalarType::Array(Box::new(ScalarType::String)), 3948;
```

Comment:
We match the oids from Postgres, it looks like these should be 394 and 376 respectively ([postgres](https://github.com/postgres/postgres/blob/master/src/include/catalog/pg_proc.dat#L1642-L1647)).

Also, I think the second argument in both of these cases should be `String`? I think even with the type of `String` you should be able to pass `NULL` as an argument.

Reply:
done! ty!